helm/reana/templates/reana-server.yaml

```diff
@@ -77,15 +80,13 @@ spec:
             value: {{ .Values.quota.default_disk_limit | default 0 | quote }}
           - name: REANA_KUBERNETES_JOBS_MEMORY_LIMIT
             value: {{ .Values.kubernetes_jobs_memory_limit | default "4Gi" }}
+          {{- $pathsDict := dict -}}
+          {{- range $workspace_paths := .Values.workspaces.paths }}
+          {{- $workspace_path := split ":" $workspace_paths }}
+          {{- $_ := set $pathsDict $workspace_path._0 $workspace_path._1 }}
+          {{- end }}          
```

Comment:
I noticed one more problem - we are loosing the order of paths since we are converting the list to dict here. 
If workspace paths have these values:
```yaml
workspaces:
  paths:
    - /var/reana:/var/reana
    - /var/eos:/eos
```
then the default value currently will be `/eos` which is wrong. It should be the first one from this list.
I guess the easiest way to solve it would be to pass a list here:
```yaml
  - name: WORKSPACE_PATHS
    value: {{ .Values.workspaces.paths | toJson | quote }}
```
 and to convert into dict in `reana-commons`

Reply:
Why is it wrong that `/eos` is the default path? I thought the idea was to have the first in the list different from `/var/reana` as the default, or do you think of allowing the admin to choose (my choosing the order) is a better idea?

Reply:
It was true before we explicitly included `/var/reana:/var/reana` to the list. Imagine that administrator would like to include `/var/eos:/eos` as an available workspace path without changing the default value. It would not be possible, since the current code would assume the new default value is `/eos` rather than `/var/reana`. If we take the first value from the list as a default one, it's up for administrator to decide which one to put first